lazy_tensor_core/test/test_optimize.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+
+from lazy_tensor_core import _LAZYC
+_LAZYC._ltc_init_ts_backend()
+from lazy_tensor_core.core.optimize import optimize
+import torch
+from torch import nn
+import dis
+import inspect
+from torch import fx
+
+class ModuleConstScale(nn.Module):
+    def __init__(self):
+        super(ModuleConstScale, self).__init__()
+
+    def forward(self, a):
+        return a * 2
+
+class ModuleSub(nn.Module):
+    def __init__(self):
+        super(ModuleSub, self).__init__()
+
+    def forward(self, a, b):
+        return a - b
+
+class ModuleAddcmul(nn.Module):
+    """
+    addcmul function takes a at::Scalar which results in a special TSData containing a Scalar rather than a Tensor.
+    """
+    def __init__(self):
+        super(ModuleAddcmul, self).__init__()
+
+    def forward(self, a, b, c):
+        return torch.addcmul(a, b, c, value=5)
+
+class ModuleReturnMulti(nn.Module):
+    def __init__(self):
+        super(ModuleReturnMulti, self).__init__()
+
+    def forward(self, a, b):
+        return (b + 1, a - 1)
+
+# The default fx tracer will convert torch.randn to a constant.. We may need
+# a custom tracer.
+# class ModuleEagerTensor(nn.Module):
```

Comment:
what is the point of this test, to see if 'randn' call can be traced as a random generator rather than burning the result of the first call into the trace?

Reply:
I was planning to use this test to cover the case that an eager tensor is created inside a Fx graph. But this behavior can not be reproduced by the default Fx tracer so far.